types/react-relay/test/react-relay-tests.tsx

```diff
@@ -5,17 +5,19 @@ import * as React from 'react';
 import { Environment, Network, RecordSource, Store, ConnectionHandler } from 'relay-runtime';
 
 import {
-    graphql,
     commitMutation,
     createFragmentContainer,
     createPaginationContainer,
     createRefetchContainer,
-    requestSubscription,
+    graphql,
```

Comment:
I just sorted the imports alphabetically

Reply:
That's fine, prettier was added to DT recently-ish 👍